src/index.ts

```diff
@@ -178,15 +179,21 @@ export default class TiledResource extends Resource<ITiledMap> {
  * Handles parsing of JSON tiled data
  */
 var parseJsonMap = (data: ITiledMap): ITiledMap => {
-   
+
    // Decompress layers
    if (data.layers) {
       for (var layer of data.layers) {
 
          if (typeof layer.data === "string") {
 
             if (layer.encoding === "base64") {
-               layer.data = decompressors.decompressBase64(<string>layer.data, layer.encoding);
+               layer.data = decompressors.decompressBase64(
+                  <string>layer.data,
+                  layer.encoding,
+                  typeof layer.compression !== "undefined"
```

Comment:
Any objections to using `layer.compression || ""` ?

Reply:
That's also a way, I'll change it.
I personally prefer the check against undefined.